### PR TITLE
travis: Remove coala checks

### PR DESCRIPTION
**Pull request by nkprince007**
- Use GitMate CI instead of ``coala --ci``

### CODE REVIEW

**Comment by nkprince007** [comment]
Hi. Thanks for contributing to UIP! Please follow 

This message was posted by [GitMate.io](https://gitmate.io) with timestamp signature `Hi. Thanks for contributing to UIP! Please follow :1eLnub:rL6yP7yDx4b013Y17lyXghubOc0`

**Comment by nkprince007** [comment]
ack https://github.com/NITDgpOS/UIP/pull/378/commits/9af472b065c8e9b0e8f148583a7bde7199b7e4bb

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,6 @@ before_script:
 
 script:
   - make install
-  - make lint
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
         dbus-launch pytest;
```
Comment (abhsag24) [inline .travis.yml]: can we have a comment here? so that we don't go over that conversation again?

**Comment by abhsag24** [comment]
ack 9af472b

File: .travis.yml (hunk 1 above)
Comment (nkprince007) [reply .travis.yml]: Sure.

**Comment by nkprince007** [comment]
ack 
https://github.com/NITDgpOS/UIP/pull/378/commits/0d2c8435a7ece9d0637ef8c451a974f0da60a8dc

**Pull request merged**